tests/charts/test_scheduler.py

```diff
@@ -304,6 +304,38 @@ def test_logs_persistence_changes_volume(self, log_persistence_values, expected_
 
         assert {"name": "logs", **expected_volume} in jmespath.search("spec.template.spec.volumes", docs[0])
 
+    def test_scheduler_securityContexts_are_configurable(self):
+        docs = render_chart(
+            values={
+                "scheduler": {
+                    "securityContexts": {
+                        "pod": {
+                            "fsGroup": 1000,
+                            'runAsGroup': 1000,
+                            'runAsNonRoot': "true",
+                            'runAsUser': 1000,
+                        },
+                        "container": {
+                            "allowPrivilegeEscalation": "false",
+                            'readOnlyRootFilesystem': "true",
+                        },
+                    }
+                },
+            },
+            show_only=["templates/scheduler/scheduler-deployment.yaml"],
+        )
+        assert "false" == jmespath.search(
+            "spec.template.spec.containers[0].securityContext.allowPrivilegeEscalation", docs[0]
+        )
+        assert "true" == jmespath.search(
+            "spec.template.spec.containers[0].securityContext.readOnlyRootFilesystem", docs[0]
+        )
```

Comment:
```suggestion
        assert {"allowPrivilegeEscalation": False,  "readOnlyRootFilesystem": True} == jmespath.search(
            "spec.template.spec.containers[0].securityContext", docs[0]
        )
```

Might be cleaner to check the whole thing in 1 go (probably need some formatting, but you get the idea).